container-images/scripts/build_llama_and_whisper.sh

```diff
@@ -270,17 +269,16 @@ main() {
   set -ex
 
   local containerfile="$1"
-  local install_prefix
+  install_prefix=$(set_install_prefix)
```

Comment:
I had been trying to keep a mantra, scope all variable in this script, even if it's local in main.

I'm actually curious why we needed to change the install_prefix stuff.

This change is fine, I just wonder why

Reply:
Not sure it is necessary. I will add it back. I was trying to get the pip installs correct and they were not using the install_prefix, but I think it should work with local now.